app/controllers/users/confirmations_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Users
+  class ConfirmationsController < Devise::ConfirmationsController
+    # GET /resource/confirmation/new
+    # def new
+    #   super
+    # end
+
+    # POST /resource/confirmation
```

Comment:
- [ ] Dear @mosidrum, I can see you have a lot of unused and commented-out controllers, This is not a good practice to leave out files with commented code in our project. Please consider learning up these files or adding the implementations.